### PR TITLE
appdev: static site builds

### PR DESCRIPTION
**Pull request by kamaln7**

### CODE REVIEW

File: internal/apps/builder/docker.go [hunk 1]
```diff
@@ -14,139 +14,165 @@ import (
...
+ARG output_dir
+
+COPY nginx.conf /etc/nginx/conf.d/default.conf
+COPY --from=content ${output_dir}/ /www
+`)
+
+	buildArgs = map[string]*string{
+		"app_image":   strPtr(b.AppImageOutputName()),
+		"nginx_image": strPtr(staticSiteNginxImage),
+		"output_dir":  strPtr(c.GetOutputDir()),
```
Comment (nicktate) [inline internal/apps/builder/docker.go:173]: Do we need to join this with the workspace path similar to the cnb asset path calculation?

**Review by nicktate — APPROVED** [review]
Overall this looks great, nice cleanup on the initial POC 🙇🏻 ; one small question in-line about the dockerfile-based asset path

File: internal/apps/builder/docker.go (hunk 1 above)
Comment (kamaln7) [reply internal/apps/builder/docker.go:173]: nope, `output_dir` must be an absolute path for dockerfile builds. we are missing that validation rule though—I'll add it

Comment (kamaln7) [reply internal/apps/builder/docker.go:181]: https://github.com/digitalocean/doctl/pull/1230/commits/7fa0c2a8eff9d634af4a10e30048ae004a78fbcb

**Pull request merged**